mongodb-odbc-driver/bin/start-sqlproxy.sh

```diff
@@ -71,8 +71,15 @@
         # sqlproxy also has no idea where the openssl libs are on macos spawns hosts because we decided to move them at
         # some point.
         if [ "$PLATFORM_NAME" = "macos" ]; then
-            install_name_tool -change /usr/local/opt/openssl/lib/libssl.1.0.0.dylib "$BREW_OPENSSL_PATH/libssl.1.0.0.dylib"  mongosqld
-            install_name_tool -change /usr/local/opt/openssl/lib/libcrypto.1.0.0.dylib "$BREW_OPENSSL_PATH/libcrypto.1.0.0.dylib"  mongosqld
+            curl -O https://mongo-bic-odbc-driver-resources.s3.amazonaws.com/macos/openssl-1.0.2n.zip
+            unzip openssl-1.0.2n.zip
+            chmod 777 ./1.0.2n/lib/*
+            mv ./1.0.2n/lib/* ./
+            install_name_tool -change /usr/local/opt/openssl/lib/libssl.1.0.0.dylib "@loader_path/libssl.1.0.0.dylib"  mongosqld
```

Comment:
I don't know why you seem to need to move the lib in the current directory and use this, but I believe that you did not do that for fun.

Reply:
`@loader_path` allows for current directory :/

Reply:
It would be possible to use the absolute path to the file, but this looks a bit cleaner, and this is also what we do with the shared objects we distribute (where we have less choice, also).

Reply:
I see, thanks for the explanation. I was just curious on this one.